apps/docs/src/app/core/component-docs/info-label/info-label-docs.component.html

```diff
@@ -19,7 +19,7 @@
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'filled'" [componentName]="'infolabel'">
+<fd-docs-section-title [id]="'infoLabelTextIcon'" [componentName]="'infolabel'">
 	Info Label with text and icon
 </fd-docs-section-title>
 <description> <code>fd-info-label fd-info-label--accent-color-1 fd-info-label--icon</code> modifier can be used to render a icon with text version of the Info Label. </description>
```

Comment:
The `fd-info-label`, `fd-info-label--accent-color-1`and `fd-info-label--icon`modifiers can be used to render an icon with a text version of the Info Label.